File: GFG/Move Last Element to Front of a Linked List/Move Last Element to Front of a Linked List.java
class Solution {
    public static Node moveToFront(Node head) {
        // code here
        if(head.next==null)return head;
        Node temp=head;
        while(temp.next.next!=null){
            temp=temp.next;
        }
        Node last=temp.next;
        temp.next=null;
        last.next=head;
        return last;
    }
}

File: GFG/Move Last Element to Front of a Linked List/Move Last Element to Front of a Linked List.cpp
class Solution{
public:
    ListNode *moveToFront(ListNode *head){
        
        if(head->next == NULL) return head;
        
        ListNode *temp = head;
        
        while(head->next->next!=NULL)
     
        head = head->next;
        head->next->next = temp;
        temp = head->next;
        head->next = NULL;
        
        return temp;
        
    }
};
